meta-mender-core/classes/mender-part-images.bbclass

```diff
@@ -137,6 +147,15 @@ EOF
 
     cat >> "$wks" <<EOF
 part --source rawcopy --sourceparams="file=${IMGDEPLOYDIR}/${IMAGE_LINK_NAME}.dataimg" --ondisk "$ondisk_dev" --align $alignment_kb --fixed-size ${MENDER_DATA_PART_SIZE_MB} --mkfs-extraopts='${MENDER_DATA_PART_FSOPTS}' $part_type_params
+EOF
+# added extra partitions if exists
```

Comment:
Nitpick: Should be indented four spaces.

Reply:
Actually I was talking about the comment, not the `${@get_extra_parts_wks(d)}` part below. The comment should be indented four spaces (outside here-document), the `${@get_extra_parts_wks(d)}` should not (inside here-document).